app/uk/gov/hmrc/tai/connectors/RtiConnector.scala

```diff
@@ -130,6 +130,7 @@ class RtiConnector @Inject()(
         }
       }
     } else {
+      logger.info(s"RTIAPI - call for CY+ year: $taxYear}")
```

Comment:
surround this logger with: 
`if(rtiToggle.rtiEnabled && taxYear.year >= TaxYear().next.year) {`

Reply:
otherwise you'll logg for CY+ for any year if rtiEnabled is disabled

Reply:
If you want to capture everything then I'd suggest changin the message to something more explicit like:
Not calling RTI API for year: year

Reply:
or Skip RTI API - call for year

Reply:
Done